webapp/channels/src/components/quick_switch_modal/quick_switch_modal.tsx

```diff
@@ -196,71 +192,75 @@ export default class QuickSwitchModal extends React.PureComponent<Props, State>
             );
         }
 
+        const modalHeaderText = (
+            <div className='channel-switcher__header'>
+                {header}
+            </div>
+        );
+
+        const modalSubheaderText = (
+            <div
+                className='channel-switcher__hint'
+                id='quickSwitchHint'
+            >
+                {help}
+            </div>
+        );
+
         return (
-            <Modal
-                dialogClassName='a11y__modal channel-switcher'
+            <GenericModal
```

Comment:
migrated QuickSwitchModal to use `GenericModal` and added the required parameters and css changes to make it look as it is expected.

Reply:
It looks like this adds a fade-in transition to the modal. Is there an option in `GenericModal` that'll let us remove that?